addons/website_sale/migrations/10.0.1.0/post-migration.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    cr = env.cr
+    pl_model = env['product.pricelist']
+    sql = """select pricelist_id, website_id, selectable from
+          website_pricelist_openupgrade_10"""
+    cr.execute(sql)
+    pricelists = []
+    for pricelist_id, website_id, selectable in cr.fetchall():
+        pl_model.search([('id', '=', pricelist_id)]).write({
```

Comment:
I'm thinking that this is better done with an update query:
```
UPDATE product_pricelist pp
SET website_id = wp.id,
    selectable = wp.selectable
FROM website_pricelist_openupgrade_10 wp
WHERE wp.pricelist_id = pp.id
```
(and make it with openupgrade.logged_query)